src/ports/linux/sampleapp_main.c

```diff
@@ -18,7 +18,7 @@
 #include "sampleapp_common.h"
 
 #include "osal_log.h"
-#include "osal.h"
```

Comment:
Without `#include "osal_log.h"` I will get 
```
In file included from /home/jonas/projects/profinetstack/p-net/sample_app/sampleapp_common.h:20:0,
                 from /home/jonas/projects/profinetstack/p-net/src/ports/linux/sampleapp_main.c:18:
/home/jonas/projects/profinetstack/p-net/src/ports/linux/sampleapp_main.c: In function ‘main’:
/home/jonas/projects/profinetstack/p-net/build/src/options.h:184:34: error: ‘LOG_LEVEL_DEBUG’ undeclared (first use in this function); did you mean ‘LOG_LEVEL’?
 #define LOG_LEVEL               (LOG_LEVEL_DEBUG)
                                  ^
```

Reply:
Removed osal-includes from `src/ports/linux/sampleapp_main.c` but added 
`#include "osal_log.h" ` to `src/ports/linux/pnal_sys.h`.
Is there a better way?

Reply:
I just meant that the file sampleapp_main.c uses functions from osal.h, yet this commit removes the #include of osal.h. It works anyway because it happens to be included from other headers, but it's usually a good idea to include files that you use. Include osal_log.h as well if you use that.

Reply:
Made sure that relevant includes are in the files.
Done